index.html

```diff
@@ -633,10 +633,23 @@ <h2>Core Data Model</h2>
 
       <p>
 The following sections outline core data model concepts, such as <a>claims</a>,
-<a>credentials</a>, and <a>presentations</a>, which form the foundation of this
+<a>credentials</a>, <a>presentations</a>, <a>verifiable credentials</a>, and
+<a>verifiable presentations</a>, which form the foundation of this
 specification.
       </p>
 
+      <p class="note"
+        title="Not every abstract concept is realized in implementations">
+Implementers might note that while there are a number of abstract concepts
+described in this section, such as <a>credential</a> and <a>presentation</a>,
+those concepts are not always concretely realized in implementations. These
+abstract concepts are provided as a mechanism to document a helpful mental model
```

Comment:
Sorry but I do not see how the abstract concepts of credential and presentation provide a helpful mental model. Precisely what are these abstract concepts, and why are they different from the concept of a verifiable credential or verifiable presentation. 
OTOH, I can see that the abstract concepts of credential and presentation are imaginary tokens that may or may not be verifiable, but by adding the adjective verifiable to them, we can see that they are verifiable and are therefore a subclass of the more general token.

Reply:
Perhaps it would be better to simply purge all mention of the terms credential and presentation and use verifiableCredential and verifiablePresentation throughout?

Reply:
> Sorry but I do not see how the abstract concepts of credential and presentation provide a helpful mental model. Precisely what are these abstract concepts, and why are they different from the concept of a verifiable credential or verifiable presentation.

This section attempts to explain that. They are abstract in that the thing isn't realized in an implementation... just like the concept of an "entity" or "repository" or a "subject" is not realized in an implementation. They are abstract (in the computer science sense of the word) because while they provide a way to think about the data model... they're not directly realized as instances. 

So, the concept of a "credential" is something that's not secured... and we don't have any normative statements about those things in this specification. The only normative statements we have are around "verifiable credentials"... because making normative statements around "credentials" doesn't really buy us anything useful.

> Perhaps it would be better to simply purge all mention of the terms credential and presentation and use verifiableCredential and verifiablePresentation throughout?

There are 504 occurrences of the word "credential" in the specification, and 79 instances of the word "presentation" in the specification. I tried going through and adding "verifiable" to the beginning of the 583 occurrences and stopped at about 100+ into the effort. Many of the sentences that we have written about just "credentials" really are about just "credentials"... not "verifiable credentials". We distinguish between the two carefully in the specification. IOW, I tried doing what you said above and it didn't go well. 

There is a difference between a credential (not secured) and a verifiable credential (secured)... and it's useful to make that distinction in the specification (and we do, in lots of places).

Reply:
I agree that there is a difference a credential (not secured) and a verifiable credential (secured) so we should describe what this difference is in the specification. You said "we don't have any normative statements about those things (i.e. credentials) in this specification". Maybe this is where we are going wrong? Because we can say that a credential MUST have a subject object and an issuer object and an `@context` and a verifiableCredential MUST have a type, an ID, an internal or external proof and MAY have validFrom, validUntil, ToU, etc. We can easily make the concept of credential concrete if we want to, and I think this will strengthen the model and the description.